src/Tests.js

```diff
@@ -73,13 +73,13 @@ function* readAllCharacteristics(device: Device): Generator<*, boolean, *> {
         yield put(log('Found characteristic: ' + characteristic.uuid));
         if (characteristic.isReadable) {
           yield put(log('Reading value...'));
-          var c = yield call([characteristic, characteristic.read]);
-          yield put(log('Got base64 value: ' + c.value));
+        //   var c = yield call([characteristic, characteristic.read]);
```

Comment:
why is it commented out instead of removed?

Reply:
Read/write operation on Descriptors and Characteristics are not yet implemented, so the test failed. They'll be gradually reintroduced as we roll out next features.